docs/ai/conceptual/zero-shot-learning.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Zero-shot and few-shot learning"
+description: "Learn the use cases for zero-shot and few-shot learning in prompt engineering."
+author: catbutler
+ms.author: [your Microsoft alias or a team alias]
+ms.service: [the approved service name]
+ms.topic: concept-article #Don't change.
+ms.date: [mm/dd/yyyy]
+
+#customer intent: As a .NET developer, I want to understand how zero-shot and few-shot learning techniques can help me improve my prompt engineering.
+
+---
+
+# Zero-shot and few-shot learning
+
+This article explains zero-shot learning and few-shot learning in .NET prompt engineering, including their primary use cases.
+
+GPT model performance benefits from [prompt engineering](prompt-engineering-in-dot-net.md), the practice of providing instructions and examples to a model to refine its output. Zero-shot learning and few-shot learning are techniques that you can use when providing examples.
+
+With zero-shot learning, you include prompts but not verbatim completions (you can include completions that only consist of cues). This means zero-shot learning relies entirely on the model's existing knowledge to generate responses, making it the most cost-effective option for prompt engineering. However, it doesn't add to the model's knowledge.
```

Comment:
I'd consider mentioning that the cost savings comes from using less tokens in the input, and any other specifics on how this saves $

Reply:
I'll ask the FTEs for any other cost savings - token reduction is the only explicit one I found but there is implication that some savings come from the model not updating its knowledge.